README.md

```diff
@@ -124,9 +108,9 @@ This port attempts to mirror the source Sass files as closely as possible in ord
 
    Note: The plugins are included using the [`@plugin`](http://lesscss.org/features/#plugin-atrules-feature) at-rule instead of as arguments to the `lessc` CLI. This was intentionally done since most Less GUI compilers don’t allow you to customize the command-line arguments.
 
-0. **Maps** Less has no _native_ concept of maps, which are used extensively in the Bootstrap Sass files. They can be emulated, however, by using a comma-separated list of space-separated lists, which is what is done in this port.
+0. **Maps** Less has native maps as of Less 3.5, but map keys must follow rules of property names (e.g. keys can't start with a number). Sass maps don't require those rules, so in this port, maps are emulated by using a comma-separated list of space-separated lists.
```

Comment:
@matthew-dean So wait, does this break in 3.8? → https://github.com/calvinjuarez/bootstrap-less-port/blob/less3.7/less/_variables.less#L23  It didn't seem to have trouble compiling when I first tried it.  (I'll have to try it out.)

Reply:
Update: that map's not currently used anywhere, so that variable would never need to be resolved.  :/

Reply:
Update again, the `@sizes` map _is_ used (with `each()` and works fine).

Reply:
Wait so is there still a question?

Reply:
I guess no, just that Less maps don't _really_ enforce any requirements on properties.  `@grays[900];` works just fine, for example.

https://gist.github.com/calvinjuarez/7da0db15c421afcca1fc53e382936cce

Reply:
Interesting. I guess I assumed the parser followed CSS rules in that regard.

Reply:
I've done a PR to Microsoft for VSCode, so it should no longer flag:

a) Numeric properties (since they're apparently allowed in Less)
b) Mixin / DR lookups

https://github.com/Microsoft/vscode-css-languageservice/pull/135